### PR TITLE
Run3 tt+X (TOP) cards (Madgraph5_aMC@NLO)

### PR DESCRIPTION
**Pull request by Cvico**
Dear experts, 

I'm a MC contact for the TOP group. We are starting to work on the Run3 TOP MC
requests, and thus we would like to have the cards included in this PR reviewed and, 
if possible, merged into the master branch.

This PR includes:
-  Slightly modified version of the cards used in Run2 tt+X production. Modifications include:
        - Center-of-mass energy adjusted to 6.8 TeV per beam.
        - Typo **nevents_for_max_weigth** has been fixed in all madspin cards.

Please let me know if you have any comment or question.

Best regards,
Carlos Vico

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#switch to non-diagonal ckm matrix if relevant
+#import model loop_sm-no_b_mass
+
+#include b quark in proton and jet definition for consistent 5 flavour scheme treatment
+define p = p b b~
+define j = j b b~
+define part =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+generate p p > t t~ a, t > b part part, t~ > b~ part part  @1
+add process p p > t t~, t > b part part a, t~ > b~ part part  @2
+add process p p > t t~, t > b part part , t~ > b~ part part a  @3
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_proc_card.dat:11]: have you checked the overlap between line 9 and line10/11

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_run_card.dat:28]: change "250000 = nevents" to a small number, 10000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+#import model loop_sm-no_b_mass
+
+#include b quark in proton and jet definition for consistent 5 flavour scheme treatment
+define p = p b b~
+define j = j b b~
+define partl =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+define partq =  u c d s u~ c~ d~ s~
+generate p p > t t~ a, t > b partl partl, t~ > b~ partq partq  @1
+add process p p > t t~, t > b partl partl a, t~ > b~ partq partq  @2
+add process p p > t t~, t > b partl partl , t~ > b~ partq partq a  @3
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_proc_card.dat:12]: same here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_run_card.dat:28]: change "250000 = nevents" to 10000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_proc_card.dat [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+#import model loop_sm-no_b_mass
+
+#include b quark in proton and jet definition for consistent 5 flavour scheme treatment
+define p = p b b~
+define j = j b b~
+define partl =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+define partq =  u c d s u~ c~ d~ s~
+generate p p > t t~ a, t > b partq partq, t~ > b~ partl partl  @1
+add process p p > t t~, t > b partq partq a, t~ > b~ partl partl  @2
+add process p p > t t~, t > b partq partq , t~ > b~ partl partl a  @3
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_proc_card.dat:12]: same here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_run_card.dat [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_run_card.dat:28]: change "250000 = nevents" to "10000 = nevents"

**Comment by Cvico** [comment]
Hi,

Thank you very much for the comments. I think with this PR I have addressed all of them.

Regarding the ttgamma samples, indeed those cards were outdated. I accidentally copied an older
version of the cards that indeed was shown to introduce double counted diagrams because of overlapping
lines 9,10 and 11 in the process card. I've included the correct ones now, and removed the old ones from the
13p6TeV folder so that they are not used by mistake.

Best regards,
Carlos

**Comment by sihyunjeon** [comment]
TTW, TTZ, TTG is in the list of central run3 sample production plans.

can you check this with gen convenors

**Comment by menglu21** [comment]
@sihyunjeon thanks for the reminder, i overlooked it. @Cvico those samples will be handled centrally, so we will put this PR on hold and make them consistent with the cards used in central production before merging.

**Pull request closed without merging**